utils/validationRules/Replies/getSingleReplyVoteSchema.js

```diff
@@ -0,0 +1,20 @@
+const Joi = require("@hapi/joi");
+/**
+ * Schema RepliesValidationRules - GET /comments/{commentId}/replies/{replyId}/votes:
+ */
+
+const getSingleReplyVoteSchema = {
```

Comment:
Your name is not a problem but just to align it with the other names please update it to this
```suggestion
const getReplyVotesSchema = {
```